plugins/woo-ai/src/utils/productData.ts

```diff
@@ -7,6 +7,7 @@ import { getTinyContent, getPostId } from '.';
 const isElementVisible = ( element: HTMLElement ) =>
 	! ( window.getComputedStyle( element ).display === 'none' );
 
+// TODO: This appears to be returning category IDs instead of names?
```

Comment:
If it's not working properly, I'm in favor of just removing it and re-adding it later when we can properly handle it.

Reply:
See my [other](https://github.com/woocommerce/woocommerce/pull/38930#discussion_r1251150706) comment.